src/Selenium2Library/locators/elementfinder.py

```diff
@@ -192,7 +205,10 @@ def _element_matches(self, element, tag, constraints):
         if not element.tag_name.lower() == tag:
             return False
         for name in constraints:
-            if not element.get_attribute(name) == constraints[name]:
+            if isinstance(constraints[name], list):
+                if element.get_attribute(name) not in constraints[name]:
+                    return False
+            elif not element.get_attribute(name) == constraints[name]:
```

Comment:
I agree but I thought you Python guys did that (based on the line removed above). Change made! :)